### DISCUSSION TITLE
posts/109

### DISCUSSION THREAD

**Discussion by giscus** [biud436/blog-front#48] [Announcements]
# posts/109

NestJS 10이 나왔고, 큰 변화는 SWC 사용 가능 정도로 보여진다.

실무에서는 파일도 많고 Windows 상에선 paths alias 설정이 동작하지 않는 문제 때문에 

https://blog.biud436.com/posts/109

**Comment by JeongYire** [comment]
SWC가 대단하긴 하군요 역시 빠른게 최고입니다

**Reply by biud436** [reply]
👍
